source/docs/software/wpilib-tools/wpical/index.rst

```diff
@@ -44,10 +44,21 @@ Example:
 
 Camera Calibration
 ------------------
-To measure the distance between tags in a video, you need the camera's intrinsics to account for things like distortion. WPIcal allows users to upload a video file to calibrate the camera in the tool, or upload a JSON file with your camera's intrinsics
+To measure the distance between tags in a video, you need the camera's intrinsics to account for things like distortion. WPIcal allows users to upload a video file to calibrate the camera in the tool, or upload a JSON file with your camera's intrinsics.
+
+The best way to obtain videos like this is with a cell phone. Users can then transfer these videos to the computer running WPIcal for processing.
 
 In-App Calibration
 ^^^^^^^^^^^^^^^^^^
+
+Linux Users Only:
+*****************
+Linux users must transcode all videos to MJPEG codec without an audio stream in an .avi file before uploading to WPIcal. This can be done with an ffmpeg command:
+``ffmpeg -i <video_file.mp4> -f avi -c:v mjpeg -b:v 4000k -an <video_file.avi>``
```

Comment:
Note that `-b:v 4000k` is totally arbitrary

Reply:
Do you think this is a good default?

Reply:
I have no idea, I just stole it from stackoverflow

Reply:
The .avi files I transcoded had bitrates exceeding 10 Mbit/s and were still noticeably grainy. The bitrate parameter should either be left out or increased to at 20 Mbit/s (arbitrary, but seems like a good minimum). A note on increasing bitrate might be good too.